lib/handlers/domHandler.js

```diff
@@ -63,7 +64,7 @@ async function assertBoundingBox(e) {
 async function getBoxModel(e) {
   return await dom
     .getBoxModel({
-      nodeId: e,
+      nodeId: isObject(e) ? e.nodeId : e,
```

Comment:
Can we use `isElement` instead of `isObject`.
`nodeId: isElement(e) ? e.get() : e,`